index.src.html

```diff
@@ -1704,6 +1717,9 @@
 
 dictionary SvcOutputMetadata {
   unsigned long temporalLayerId;
+  unsigned long spatialLayerId;
+  unsigned long long frameId;
```

Comment:
Could we have frameId and dependencies directly in EncodedVideoChunkMetadata? They're general fields, which make sense and are generally useful even in configs without multiple temporal/spatial layers.

Reply:
Yes, we can do that.